README.md

```diff
@@ -1,160 +1,111 @@
-
-🚫Replace text below with a description of the API
-
-Water's like me. It's laaazy ... Boy, it always looks for the easiest way to do things A little happy sunlight shining through there. Let all these little things happen. Don't fight them. Learn to use them. Even the worst thing we can do here is good.
+#### [Back end](https://github.com/Lambda-School-Labs/neighborhood-chef-be) built using:
 
-## 2️⃣ Payment API here
+#### NodeJS/ExpressJS
 
-🚫Replace text below with a description of the API
+- embraces javascript-based programming
+- when grouped with basic middlewares (Helmet, CORS, Compression, Multer, JSON), provides robust server structure
+- easily scalable
 
-This is the way you take out your flustrations. Get away from those little Christmas tree things we used to make in school. Isn't it fantastic that you can change your mind and create all these happy things? Everything's not great in life, but we can still find beauty in it.
+#### GraphQL Query language
 
-## 3️⃣ Misc API here
+- simple, easy to read queries and data retrieval
+- very efficient queries
+- predictable in what will be provided
 
-🚫Replace text below with a description of the API
+#### PostgreSQL Database
 
-You can do anything your heart can imagine. In life you need colors. This is where you take out all your hostilities and frustrations. It's better than kicking the puppy dog around and all that so. I'm sort of a softy, I couldn't shoot Bambi except with a camera. Trees get lonely too, so we'll give him a little friend. We'll lay all these little funky little things in there.
+- fully capable and diverse with many different index types
+- open source source code for visibility
+- allows full support for SSL and other security measures
+- atomic and consistent
 
-## 3️⃣ Misc API here
+#### Back end deployed using AWS Amplify
 
-🚫Replace text below with a description of the API
-
-When you do it your way you can go anywhere you choose. Let your heart take you to wherever you want to be. If I paint something, I don't want to have to explain what it is. A tree needs to be your friend if you're going to paint him. That's a son of a gun of a cloud. Even the worst thing we can do here is good.
-
-## 3️⃣ Misc API here
+# APIs
 
-🚫Replace text below with a description of the API
+## Okta Authentication API
 
-Volunteering your time; it pays you and your whole community fantastic dividends. Maybe there's a happy little waterfall happening over here. You can spend all day playing with mountains. We don't have to be committed. We are just playing here. You have freedom here. The only guide is your heart. It's cold, but it's beautiful.
+  Robust and trusted authentication platform that is easily integrated into React front end as well as NodeJS backend frameworks
 
-# 3️⃣ Environment Variables
+# Environment Variables
 
 In order for the app to function correctly, the user must set up their own environment variables. There should be a .env file containing the following:
```

Comment:
Might want to use `dotenv-safe` node library and reference the `.env.example` file it requires.
https://github.com/rolodato/dotenv-safe